nethserver-ntopng.spec

```
@@ -41,7 +41,8 @@ cp -a logo.png %{buildroot}/usr/share/cockpit/%{name}/
 cp -a %{name}.json %{buildroot}/usr/share/cockpit/nethserver/applications/
 cp -a api/* %{buildroot}/usr/libexec/nethserver/api/%{name}/
 
-%{genfilelist} %{buildroot} --dir /var/run/ntopng "%attr(0755,nobody,nobody)"  > %{name}-%{version}-filelist
+%{genfilelist} %{buildroot} --dir /var/run/ntopng "%attr(0755,nobody,nobody)"  /
```

Fix Bash line continuation

```suggestion
%{genfilelist} %{buildroot} --dir /var/run/ntopng "%attr(0755,nobody,nobody)"  \
```